RELEASE_NOTES

```diff
@@ -1,3 +1,15 @@
+=======================================================================
+amazon-cloudwatch-observability v2.0.1 (2024-08-23)
+========================================================================
+New Features:
+* Adding support for NodeJS auto instrumentation for Application Signals (#91)
```

Comment:
we should probably mention my windows daemonset change: https://github.com/aws-observability/helm-charts/commit/802c0c494367d8dd14b580997f47555450638e38

Reply:
updated